src/wizards/tokenSwapDealWizard/stages/submitStage/submitStage.scss

```diff
@@ -0,0 +1,191 @@
+  color: $Secondary05;
+  font-size: 18px;
+  align-self: center;
+  justify-self: end;
+  background-position: center;
+  background-size: cover;
+}
+
+$submitCardWidth: 645px;
+
+.submitStageSectionContentContainer {
+  font-family: Inter;
+
+  .stageSectionContent {
+    margin-bottom: 30px;
+
+    &:last-child {
+      margin-bottom: inherit;
+    }
+
+    .subtext {
+      color: $Neutral04;
+    }
+  }
+
+  .submitContentLabel {
+    @include commonContentLabel;
+    font-size: $font-size3;
+    line-height: 33px;
+  }
+
+  .submitContentLabel-sm {
+    @include commonContentLabel;
+    font-size: $font-size2;
+    line-height: 22px;
+  }
+
+  .submitCard {
+    width: $submitCardWidth;
+
+    pcard {
+      display: block;
+      padding: 20px;
+    }
+
+    &.submitDaoCard {
+      .submitContent {
+        margin: unset;
+      }
+    }
+
+    &.submitTermsCard {
+      margin-top: 30px;
+
+      .subtext {
+        font-size: $font-size3;
+      }
+    }
+
+    &.submitDaoCard {
+      .submitDaoTokenCard {
+        pcard {
+          background: $BG01;
+          margin-top: 20px;
+        }
+
+        .tokenName {
+          display: flex;
+          align-items: center;
+          gap: $spacing-normal;
+          font-weight: $font-bold;
+          font-size: $font-size1;
+          line-height: 20px;
+        }
+
+        .detailsTable {
+          display: grid;
+          grid-template-columns: 1fr 2fr;
+          gap: 15px;
+          row-gap: 20px;
+          width: 100%;
+          border-spacing: $spacing-normal;
+
+          .detailsCell {
+            max-width: 130px; // TODO: experimented, until it looks good --> Should ask Design
+
+            p {
+              text-overflow: ellipsis;
+              overflow-x: hidden;
+            }
+
+            p:first-child {
+              color: $Secondary02;
+              margin-bottom: $spacing-small;
+              display: flex;
+              align-items: center;
+              gap: $spacing-medium-small;
+              font-weight: $font-bold;
+            }
+          }
+
+          &.detailsTableInViewMode {
+            grid-template-columns: 1fr 1fr 1fr 1fr;
+            margin: 30px 0;
+
+            .detailsCellAmount {
+              grid-column: span 3;
+            }
+          }
+        }
+      }
+    }
+
+    .submitCardContent {
+      .submitCardHeader {
+        font-size: $font-size3;
+        font-weight: $font-bold;
+        border-bottom: 1px $Secondary05 solid;
+        margin-bottom: 18px;
+        padding-bottom: 6px;
+        line-height: 28px;
+      }
+
+      .submitContent {
+        margin-bottom: 20px;
+      }
+
+      .daoAvatar {
+        @include daoAvatar(36px);
+      }
+    }
+  }
+
+  .optionalContent {
+    color: $Neutral02;
+  }
+}
+
+
+.congratulatePopup {
+  margin-bottom: 18px;
+  max-width: 600px;
+
+  .ppopup-dialog.pcard.gradient {
+    /* Fix pink line appearing */
+    border-right-width: 7px
```

Comment:
Are you sure this still needs fixing?  Any bug in that popup should be made in the popup, not here.